docker/service.go

```diff
@@ -282,7 +282,13 @@ func (s *Service) Info(ctx context.Context, qFlag bool) (project.InfoSet, error)
 
 // Start implements Service.Start. It tries to start a container without creating it.
 func (s *Service) Start(ctx context.Context) error {
-	return s.up(ctx, "", false, options.Up{})
```

Comment:
Calling `up` here would create the container if not already created. Which is not right, as said, `Start` start the container without creating it.